### PR TITLE
Use elemental upgrade instead of cos.sh script

### PR DESCRIPTION
**Pull request by Itxaka**
Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

File: packages/installer/upgrade.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -e
 
-cos upgrade $@
+elemental --debug --logfile /tmp/elemental.log upgrade $@
```
Comment (davidcassany) [inline packages/installer/upgrade.sh:4]: Good point I forgot about the logfile in the installer and was about to for the reset command

**Review by davidcassany — APPROVED** [review]
Looks good

File: packages/installer/upgrade.sh (hunk 1 above)
Comment (Itxaka) [reply packages/installer/upgrade.sh:4]: The logfile is also picked up by the test runner if there is a failure and uploaded to github!

File: tests/installer/installer_test.go [hunk 2]
```diff
@@ -203,29 +202,19 @@ var _ = Describe("cOS Installer tests", func() {
...
+			It("fails to install with a wrong config file and strict flag", func() {
 				err := s.SendFile("../assets/broken-config.yaml", "/tmp/config.yaml", "0770")
 				By("Running the cos-installer with a broken config file")
 				Expect(err).To(BeNil())
 				By("Running the installer")
-				out, err := s.Command("cos-installer --cloud-init /tmp/config.yaml /dev/sda")
-				Expect(err).To(BeNil())
+				out, err := s.Command("cos-installer --strict --debug --cloud-init /tmp/config.yaml /dev/sda")
+				Expect(err).To(HaveOccurred())
 				Expect(out).To(ContainSubstring("Copying Active image..."))
```
[the remark this answers is not in the document]
Comment (Itxaka) [reply tests/installer/installer_test.go:212]: yeah, was trying to somehow make it work but...like we are already testing this part on elemental and its easy to test in there...should probably kill it and forget about it.

**Comment by davidcassany** [comment]
@Itxaka I think we can ignore the installer tests for this PR and fix them in follow up PRs (they are unrelated to upgrade), if the tests for the intalled system (squashfs and nonsquashfs) are green I'd merge it.

**Comment by Itxaka** [comment]
> @Itxaka I think we can ignore the installer tests for this PR and fix them in follow up PRs (they are unrelated to upgrade), if the tests for the intalled system (squashfs and nonsquashfs) are green I'd merge it.

Should be an easy fix, its only that the installer has changed the log output to use the LABEL instead of the active/passive. I can add those changes to this PR

**Pull request merged**